### DISCUSSION TITLE
Animating between stack navigators

### DISCUSSION THREAD

**Discussion by Tadejo9** [react-navigation/react-navigation#9686] [General]
Hello 👋 

I'm trying to make a transition between AuthRoutes and AppRoutes.

I've read the docs but found only animation configs for transitions between screens.

```
const Routes: React.FC = () => {
    const { isSignedIn } = useAuth();
  
    return isSignedIn ? <AppRoutes /> : <AuthRoutes />;
}
```

Above is the example where AuthRoutes and AppRoutes are separate stack navigators.

Is there a way of animating between navigators when isSignedIn changes or should i join them in one navigator as separate screens?

Have a nice day!

**Comment by satya164** [comment]
You should use a single stack navigator for them

**Comment by michaeloliverx** [comment]
I am facing this issue too, what way did you solve it?

I would prefer to have separate stack navigators as (I think?) it makes the typescript integration much easier as you don't have to nest navigators.
